airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2449,7 +2470,7 @@ components:
       properties:
         total_entries:
           type: integer
-          description: Total count for all collection items.
+          description: Count of objects in the current result set.
```

Comment:
The description of this field was very troublesome. I wrote it a little differently.

Reply:
This would affect some endpoints -> connections, variables etc. Can you take a look and see if they need changing?

Reply:
I will correct it. We were very inconsistent with this field.

Reply:
I checked everything in detail and it seems to me that everything is okay.  If the resource has no filters, this number equals the total number of objects in the database. We didn't support filters very often, so we could get a count of all objects in many cases.

Reply:
The way I think about this is that, the total entry should be equal to total results else we may end up returning no items but have a value for total entries

Reply:
This field should contain information on the number of elements on all pages with current filters. This will allow us to fetch multiple pages in parallel.